### PR TITLE
For Host only, set disconnect/disband to disband only

### LINKED ISSUE #183: Disconnect button on host when guest not connected is confusing.

### LINKED ISSUE #184: Disband button does not go away when last guest drops.
Observed when performing test 22.

### PR DESCRIPTION
**Pull request by taywrobel**
Change the functionality of the disconnect/diaband
button in the network fragment to serve as disband
at all times when you are host, and disconnect at
all times when you are guest.

Fixes #184
Fixes #183 by changing disband confirmation wording

### CODE REVIEW

**Comment by dgreenhalgh** [comment]
@theJenix

File: SoundStream/src/com/lastcrusade/soundstream/components/NetworkFragment.java [hunk 1]
```diff
@@ -150,17 +149,15 @@ public void onClick(View v) {
...
-        } else{
-            if(disconnectDisband != null){
-                setDisconnectFunction();
-            }
-        } 
+		// The only time a host should be connected is if we are a guest. In
+		// this case, it should be a disband function, otherwise (when we are
+		// host) it should be be a disconnect function.
+		if (getConnectionService() == null
+				|| getConnectionService().isHostConnected()) {
```
Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/components/NetworkFragment.java]: Please fix the spacing here...the indentation is out of whack.

**Comment by theJenix** [comment]
This looks good to me after you fix the :space_invader:  in NetworkFragment.java

**Comment by theJenix** [comment]
Looks :nine: to me.

**Comment by dgreenhalgh** [comment]
:shipit:

**Pull request merged**